### DISCUSSION TITLE
hi im getting this error please help

### DISCUSSION THREAD

**Discussion by raju-b70** [learndevopsonline/learndevopsonline#1845] [Q&A]
![Screenshot (28)](https://github.com/learndevopsonline/learndevopsonline/assets/161826606/ad541e30-8e07-4db0-9635-d28317bed63c)
![Screenshot (29)](https://github.com/learndevopsonline/learndevopsonline/assets/161826606/6a26a17b-ca57-42bf-a3f7-b9cdf2c81ba9)

**Comment by chaithanya1812** [comment]
# Hi raju, you are doing wrong !
## how many discussions you have raised for one problem,

### check once this I already mention in previous disscussion, In command you need use ansible_password=DevOps321

![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/3fecfd57-f15d-49ff-902a-269ebc47ef81)

**Reply by raju-b70** [reply]
sorry im not able to open that previous session page it shows session expired like that .thank you
